src/CBT.Core/Tasks/RestoreModules.cs

```diff
@@ -215,6 +219,15 @@ public bool Execute(string[] afterImports, string[] beforeImports, string extens
             return Execute();
         }
 
+        private string GetAssetsFileDirectory()
```

Comment:
Can we detect the case when there should be a flag file but its not there?  It might be worth giving a good error to the user that something is wrong.  I think right now everything would fail silently right?